engine/pokemon/mon_menu.asm

```diff
@@ -848,12 +848,12 @@ ChooseMoveToDelete:
 	pop af
 	ret
 
-DeleteMoveScreenAttrs:
-	db 3, 1
-	db 3, 1
-	db $40, $00
-	dn 2, 0
-	db D_UP | D_DOWN | A_BUTTON | B_BUTTON
+DeleteMoveScreen2DMenuData:
+	db 3, 1 ; cursor start y, x
+	db 3, 1 ; rows, columns
+	db $40, $00 ; flags
```

Comment:
Doesn't menu_constants.asm have values for these?

Reply:
These relate to `w2DMenuFlags1/2`. They don't have constants yet, and aren't very well documented. This is material for a different PR. Maybe open an issue about it.